specification/mysql/resource-manager/Microsoft.DBforMySQL/preview/2017-12-01-preview/examples/ServerCreate.json

```diff
@@ -18,10 +18,10 @@
         "createMode": "Default"
       },
       "sku": {
-        "name": "B_Gen4_2",
+        "name": "B_Gen5_2",
```

Comment:
Are examples the only places these values appear at? Is there associated documentation or swagger spec that needs to be updated as well?

Reply:
@praries880 I think examples is the only place where you can find these values in the REST API specs. If examples are wrong the only way to figure out proper values is to go to the portal, or use Azure CLI to create the server and then see what values are correct.

Anyway, thanks for reminding me. I did a search and found out that we could also update response values -- this PR only fixes values in the requests.

Reply:
@zikalino will you be updating the PR to fix the values in the responses as well?

Reply:
@praries880 everything is already correct, I haven't found any occurrences of Gen4 anywhere. So I think we can go ahead and merge this PR.